### ISSUE TITLE
Explain the necessity of "variant" in the configuration spec and documentation

### ISSUE THREAD

**Issue by brianredbeard** [coreos/fcct#20]
Within the documentation for the configuration [spec][spec] it calls out "variant" as being a required field.  At the same time this is not present in the published documentation (though it is shown in the [source code][source]): 

![image](https://user-images.githubusercontent.com/57542/61487433-c17f3180-a95a-11e9-9453-182c27dcd28b.png)

With simple examples like this, it would help to explain why this is specific to my use case as well as why a rendered ignition sequence[1] with the same functional content would be denied in use:

    [[0;32m  OK  [0m] Stopped target [0;1;39mTimers[0m.[   17.900163] ignition[589]: failed to fetch config: unsupported config version

[1]: Rendered ignition configuration used with a different ignition based host - 

```
{
  "ignition": {
    "config": {},
    "security": {
      "tls": {}
    },
    "timeouts": {},
    "version": "2.2.0"
  },
  "networkd": {},
  "passwd": {
    "users": [
      {
        "name": "core",
        "sshAuthorizedKeys": [
          "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDVUVfmNA8ybrjfBMJQrJS2stImcGdaiWSAdkCnVi0urbRwChnREhndrCueMyQwUeofzZuyCZAQ323QSAEussO+H5vSgCT+lM93oeN7nays5Y9l3kaj3eJtDzSeJ7e7X0Bo//BO4xevO1PqlfplwPGXJgV61O/iRYP9EtkCXbgjO40IpD3knXw6De++fqSdKe6FPdWkjQTLvR8X558UXf+C4CaP8xMLbVsH+lIWdm4yRRn/RM+qjCUTQDPMPkEZy/gsL0P0HX1Ecx0FcoXSsVcLMed3HIQFXrw+MD2gLbQ3yWvKQO+EvD47TRwlU4WcuM/Ua3cC1R/eK2LlovPih8/pS8aeSQG+BRb/UhTf6gZfDTTUyM/SuYCM9RWMT0SS9SEuyo2q9WrOOQtu6ouLJHyYZMX7jbkuzmAEJDGtvfYzSC6vvHK8rOMdJgMujnqHmV2JoUbubkivE8pcJQYcH1MzCoeNMqzzBHocJCJ/aqDPNbyselB3BeiCjgIIoI6naZBrQCFotD5mH+VC394IL36T1lfgu4gJMaPORqP4GM+DyZMcndz3LSV5zxfVJrVsamGoKAZgNlOVw+AbBf0L/+c7xfeXKT+JT/lVXdSanjhWYCcQla6oHvi+MxZO4lfGjaBXXLssxDf9+O5ikZ/5fMnffog4xJlMte12JN98idKGjw=="
        ]
      },
      {
        "name": "bharrington",
        "sshAuthorizedKeys": [
          "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDVUVfmNA8ybrjfBMJQrJS2stImcGdaiWSAdkCnVi0urbRwChnREhndrCueMyQwUeofzZuyCZAQ323QSAEussO+H5vSgCT+lM93oeN7nays5Y9l3kaj3eJtDzSeJ7e7X0Bo//BO4xevO1PqlfplwPGXJgV61O/iRYP9EtkCXbgjO40IpD3knXw6De++fqSdKe6FPdWkjQTLvR8X558UXf+C4CaP8xMLbVsH+lIWdm4yRRn/RM+qjCUTQDPMPkEZy/gsL0P0HX1Ecx0FcoXSsVcLMed3HIQFXrw+MD2gLbQ3yWvKQO+EvD47TRwlU4WcuM/Ua3cC1R/eK2LlovPih8/pS8aeSQG+BRb/UhTf6gZfDTTUyM/SuYCM9RWMT0SS9SEuyo2q9WrOOQtu6ouLJHyYZMX7jbkuzmAEJDGtvfYzSC6vvHK8rOMdJgMujnqHmV2JoUbubkivE8pcJQYcH1MzCoeNMqzzBHocJCJ/aqDPNbyselB3BeiCjgIIoI6naZBrQCFotD5mH+VC394IL36T1lfgu4gJMaPORqP4GM+DyZMcndz3LSV5zxfVJrVsamGoKAZgNlOVw+AbBf0L/+c7xfeXKT+JT/lVXdSanjhWYCcQla6oHvi+MxZO4lfGjaBXXLssxDf9+O5ikZ/5fMnffog4xJlMte12JN98idKGjw=="
        ]
      }
    ]
  },
  "storage": {},
  "systemd": {}
}
```

[spec]: https://github.com/coreos/fcct/blob/846a7981e3a8561345c6ae8edf9586c2deb435a1/docs/configuration-v1_0.md
[source]: https://github.com/coreos/fedora-coreos-docs/blob/382d2c8fdec0c07305052f61f36740b1414c91bb/modules/ROOT/pages/getting-started.adoc#generating-ignition-configs

**Comment by ajeddeloh** [comment]
Looks like the fix for the FCOS docs already landed. Added explanation in https://github.com/coreos/fcct/pull/25

**Comment by ajeddeloh** [comment]
closed via #25
